contents/blog/best-mixpanel-alternatives.md

```diff
@@ -0,0 +1,498 @@
+ <td>Track metrics at account or company level</td>
+ </tr>
+ <tr>
+ <td><strong>Realtime dashboards</strong></td>
+ <td className="text-center"><span className="text-green text-lg">✔</span></td> 
+ <td className="text-center"><span className="text-red text-lg">✖</span></td> 
+ <td>Track website activity in real time</td>
+ </tr> 
+ <tr>
+ <td><strong>Data governance</strong></td>
+ <td className="text-center"><span className="text-red text-lg">✖</span></td> 
+ <td className="text-center"><span className="text-green text-lg">✔</span></td> 
+ <td>Admin level control over event approval, tags and descriptions</td>
+ </tr> 
+ </tbody>
+</table>
+</div>
+
+![GA4](../images/blog/mixpanel-alternatives/GA4.png)
+
+### Why do companies use GA4?
+
+1. **Because it's Google:** It's not an original reason, but it holds true. Using Google Analytics makes a lot of sense for teams who rely on other Google platforms, like Google Ads and BigQuery.
+
+2. **It's powerful and free:** Likewise, Google's scale means GA4 is completely free to most small and medium-sized businesses. This, combined with strong analytical tools and the large ecosystem of GA experts to call upon, makes it a safe choice.
+
+3. **To track marketing ROI:** GA4 is predominantly used by marketing and e-commerce teams to track campaign ROI. It's also popular among large content publishers for its scalability.
+
+### How popular is GA4?
+
+Google Analytics remains the most-used analytics tool in the world by a large margin. 
+
+As of July 2023, [30.5% of the top 1 million websites](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/) run GA4 – another 35.7% were still running GA3.
+
+> ### Bottom line
+>
+> GA4 is the best Mixpanel alternative for marketing teams who want to track ROI on campaigns, and a good option for B2C and e-commerce platforms. Product teams, especially those working on B2B products, should look elsewhere.
+
+## Amplitude
+
+![amplitude](../images/blog/mixpanel-alternatives/amplitude-website.png)
+
+### What is Amplitude?
+
+Amplitude is an analytics and testing tool with a particular focus on large enterprise customers, like Ford, NBCUniversal, and Walmart. 
+
+### Key features
+
+- 📈 **Product analytics**, including funnel and retention analysis, user paths, behavioral cohorts, custom dashboards, and more.
+
+- 🧪 **A/B testing** with support for JSON payloads, primary, secondary, and counter metrics.
+
+- 💁‍♂️ **Customer data platform** that combines analytics data with third-party tools for data governance, identity resolution, and data federation.
+
+- 🤖 **AI insight builder** that generates insights based on natural language requests, like "What is my purchase conversion rate?".
+
+### Who uses Amplitude?
+
+Typical Amplitude users are:
+
+- Non-technical product managers
+- Data teams at large enterprises
+- Growth engineering and marketing teams
+
```

Comment:
Companies with heavy regulatory requirements like banking and healthcare (at least that what it seems like from their page).